src/protocolgame.cpp

```diff
@@ -916,6 +917,14 @@ void ProtocolGame::parseFollow(NetworkMessage& msg)
 	addGameTask(&Game::playerFollowCreature, player->getID(), creatureId);
 }
 
+void ProtocolGame::parseEquipObject(NetworkMessage& msg)
+{
+	uint16_t spriteId = msg.get<uint16_t>();
+	msg.get<uint8_t>();
```

Comment:
You can comment this line since it doesn't really do anything other than move the read position, which makes no difference here, but it's good to know there's one more byte being sent in this packet.